x-pack/plugins/security_solution/common/api/detection_engine/signals_migration/create_signals_migration/create_signals_migration.gen.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Initiates alerts migration API endpoint
+ *   version: 2023-10-31
+ */
+
+import { NonEmptyString } from '../../../model/primitives.gen';
+
+export type CreateAlertsMigrationRequestBody = z.infer<typeof CreateAlertsMigrationRequestBody>;
+export const CreateAlertsMigrationRequestBody = z.object({
+  index: z.array(NonEmptyString).min(1),
+  requests_per_second: z.number().int().min(1).optional(),
+  size: z.number().int().min(1).optional(),
+  slices: z.number().int().min(1).optional(),
+});
+
+export type CreateAlertsMigrationResponse = z.infer<typeof CreateAlertsMigrationResponse>;
+export const CreateAlertsMigrationResponse = z.object({
```

Comment:
The migration response can have an error as part of the `ok` response: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/create_signals_migration_route.ts#L142
Looks like this response should be a union of the error and regular responses